script.js

```diff
@@ -0,0 +1,167 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+  startTime();
+
+  let currentTab = getCurrentTabFromURL();
+
+  const triggerTabList = [].slice.call(document.querySelectorAll('#myTab a'));

```

Comment:
выглядит интересно, можно попроще думаю
```suggestion
  const triggerTabList = [...document.querySelectorAll('#myTab a')];

```